.github/workflows/template-e2e-notebook-test/action.yaml

```diff
@@ -0,0 +1,54 @@
+name: Notebook test template
+description: A composite action to setup and run example notebooks using Papermill
+
+inputs:
+  python-version:
+    required: false
+    description: Python version
+    default: "3.9"
+  notebook-input:
+    description: 'List of paths to the input notebooks, separated by commas'
+    required: true
+
+runs:
+  using: composite
+  steps:
+
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ inputs.python-version }}
+
+    - name: Install dependencies
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip
+        pip install papermill kubeflow-katib jupyter ipykernel
+        python -m ipykernel install --user --name python3 --display-name "Python 3"
+
+    - name: Setup Minikube Cluster
+      uses: medyagh/setup-minikube@v0.0.18
+      with:
+        network-plugin: cni
+        cni: flannel
+        driver: none
+        kubernetes-version: v1.29.2
+        minikube-version: 1.34.0
+        start-args: --wait-timeout=120s
+
+    - name: Setup Minikube
+      shell: bash
+      run: ./test/e2e/v1beta1/scripts/gh-actions/setup-minikube.sh true true "" "" "cmaes"
```

Comment:
Can we re-use this template for this action, but passing the appropriate values to it:
https://github.com/kubeflow/katib/blob/683608f6a61e7b10218b7084f310af42334d8e65/.github/workflows/template-e2e-test/action.yaml
We can have one more input for `notebooks` tests that triggers `run-notebook.sh`.